data/Templates/Ccda/Resource/_Encounter.liquid

```diff
@@ -19,10 +19,6 @@
             {% include 'DataType/Coding' Coding: encounter.code -%}
             {% endif -%}
         },
-        "type":
```

Comment:
Why do we remove this property?

Reply:
In the original template, the value of the `type` property is got from `encounter.code`, while it is wrong. The value of `encounter.code` is converted to 'class' property instead of `type` property.

Reply:
Got it.

Reply:
Just curious what's the difference between `type` and `class`? Do we have any references about this mapping relationship?

Reply:
By offline discussion, we will map `code` to `class` according to their semantics.